modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -277,3 +277,28 @@ def __dump_netfilter_settings(node):
         Reads current netfilter settings on the node for debugging purposes.
         """
         return str(node.account.ssh_client.exec_command("sudo iptables -L -n")[1].read(), sys.getdefaultencoding())
+
+    def restart(self, timeout_sec=180):
+        """
+        Restart ignite cluster without cleaning.
+        """
+        self.stop()
+
+        for node in self.nodes:
+            super().start_node(node)
+
+        self.logger.info("Waiting for Ignite(s) to start...")
+
+        for node in self.nodes:
+            self.await_node_started(node, timeout_sec)
```

Comment:
sync start already provides such guarantee

Reply:
nope, they `wait_until(lambda: self.alive(node), timeout_sec=10)`
the node is still in the process of starting